.env.template

```
@@ -7,18 +7,28 @@ MAX_CPU=1
+CONFIGURATION_DATABASE_USER='postgres'
+CONFIGURATION_DATABASE_PASSWORD='password'
+CONFIGURATION_DATABASE_HOST='localhost'
+
+EVENT_HISTORY_DATABASE_PORT='5432'
+EVENT_HISTORY_DATABASE='event_history'
+#EVENT_HISTORY_DATABASE_CERT_PATH=''
+EVENT_HISTORY_DATABASE_USER='postgres'
+EVENT_HISTORY_DATABASE_PASSWORD='password'
+EVENT_HISTORY_DATABASE_HOST='localhost'
+
+EVALUATION_DATABASE_PORT='5432'
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)